lib/content-services/src/lib/aspect-list/aspect-list.component.spec.ts

```diff
@@ -165,27 +163,40 @@ describe('AspectListComponent', () => {
         });
 
         it('should return true when same aspect list selected', () => {
+            fixture.detectChanges();
             expect(component.hasEqualAspect).toBe(true);
         });
 
         it('should return false when different aspect list selected', () => {
+            fixture.detectChanges();
             component.clear();
             expect(component.hasEqualAspect).toBe(false);
         });
 
         it('should show all the aspects', async () => {
+            fixture.detectChanges();
             expect(await loader.hasHarness(MatExpansionPanelHarness.with({ selector: '#aspect-list-FirstAspect' }))).toBe(true);
             expect(await loader.hasHarness(MatExpansionPanelHarness.with({ selector: '#aspect-list-SecondAspect' }))).toBe(true);
         });
 
         it('should show aspect id when name or title is not set', () => {
+            fixture.detectChanges();
             const noNameAspect = fixture.nativeElement.querySelector('#aspect-list-cst-nonamedAspect .adf-aspect-list-element-title');
             expect(noNameAspect).toBeDefined();
             expect(noNameAspect).not.toBeNull();
             expect(noNameAspect.innerText).toBe('cst:nonamedAspect');
         });
 
+        it('should not show aspect if it is excluded aspect', () => {
+            component.excludedAspects = ['cst:nonamedAspect'];
+
+            fixture.detectChanges();
```

Comment:
Just a thought maybe we can leave `fixture.detectChanges();` call in beforeEach and just call it again only in this test case? I know that it will be called twice but then you wouldn't have to manually call it in every other test case, what do you think?

Reply:
I can't call it twice because it is in ngOnInit which be triggered for first execution of detectChanges. But I have prepared now two separate describe blocks so now I can use detectChanges in beforeEach for rest tests.